wallet/test/run-psbt_fixup.c

```diff
@@ -0,0 +1,188 @@
+	"70736274FF01007D020000000147E2A8CCB2D1DFC409B5F9222A13CC22825D46A4966B955B67198190AC4C9901010000000080982680029D0C0A00000000002200201727604C9DFE8177138914760909848F161DE9378BB581AA514DEFD26DEA4371BA0D1C00000000001600141D0217D699F9CFB94C5052A3133B33CA5C2A1BCB0B49D0200001012BA02526000000000022002042B5EF1E1BC56C5E2576253B21CC27EFE926DB2E672667832178DF1807D1B5A4220203E51E3C2BAA34316EF01618437B905C217F234DAB7200899F2E3BFF66190DA98A473044022070963A24CA8FBEB7FED9CA8A2F260DAC36784A3260B11556074804993ED1D8C0022046D4D41C60E9179147016FEC3EC12C0FC464F21F3C3E0D0B070C2ACA1D98D8E80101030401000000010547522103E51E3C2BAA34316EF01618437B905C217F234DAB7200899F2E3BFF66190DA98A2103F2D9ED6C61D7FBE813CE7C2E3485ADD3B97E1E26F63F996C4AA2C8655439DC6E52AE220603F2D9ED6C61D7FBE813CE7C2E3485ADD3B97E1E26F63F996C4AA2C8655439DC6E084BC3C06D00000000220603E51E3C2BAA34316EF01618437B905C217F234DAB7200899F2E3BFF66190DA98A089A2E9D88000000000001014D632103B0CAACE6204A2CD80F003D2098996A7C705051A6D5E8557F2188EF9A453DF26A67029000B2752102236102A702BE93F55171673FE54AF3FB37B327D164B15D27EA83B103EE49BFBD68AC0000",
+	"70736274FF01005E020000000124AAD697740925E0723F499A42CE437E9C94FA3FC2E40554EE0B279FC34CC14701000000002A8BD38001B8DE4F00000000002200207EEAE0C3008987EB05E6FA6331D83FB8C91A182F7330586769794791C02EA66ACA464B200001012B00EA4F00000000002200208C5C882EB2E4E98F3E099C24493684263C4D25795CE753C4CCE0696BD6430959220203966DFB3900DB95001B8E19A90A4ABEE6F2A87953BE7ABA93963B0A5813BEFEB0483045022100AAFD56E98DEE620B4549BF5D998176FD5324C9171F314F4DA02C928A10EA8CCD022031D91F26CE0E599601CF1AA0317A68A32D19B1BB0D590163E271A5FC5F56C53F010103040100000001054752210362CA871F1041F5383F12B984F75A8EDB3C0CF2ECD397C6071E08B8C43E0FC02D2103966DFB3900DB95001B8E19A90A4ABEE6F2A87953BE7ABA93963B0A5813BEFEB052AE22060362CA871F1041F5383F12B984F75A8EDB3C0CF2ECD397C6071E08B8C43E0FC02D08B38CC09300000000220603966DFB3900DB95001B8E19A90A4ABEE6F2A87953BE7ABA93963B0A5813BEFEB008AF472E7E000000000001014D63210271CA81B4D46590CEBDC41C8DDBC795D3AECF611FDBD1674C570FB7C216C6947867027502B27521023F097BD253DC4AD9C534026662A01A5A7528F0C4A1803D7533B7718FC38CAB1E68AC00",
+};
+
+int main(int argc, char *argv[])
+{
+	struct wally_psbt *psbt;
+	u8 *data;
+
+	common_setup(argv[0]);
+	chainparams = chainparams_for_network("bitcoin");
+
+	for (size_t i = 0; i < ARRAY_SIZE(psbts); i++) {
+		data = tal_hexdata(tmpctx, psbts[i], strlen(psbts[i]));
+		psbt = psbt_from_bytes(tmpctx, data, tal_bytelen(data));
+		if (psbt) {
```

Comment:
do a quick check that both branches are taken at least once to check noop and real fixup behavior?

Reply:
I used to have fprintf in both, but I'll re-add an assertion that 7 get fixed up.